recipes/libxcrypt/meta.yaml

```diff
@@ -0,0 +1,73 @@
+{% set name = "libxcrypt" %}
+{% set version = "4.4.28" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/besser82/libxcrypt/releases/download/v{{ version }}/libxcrypt-{{ version }}.tar.xz
+  sha256: 9e936811f9fad11dbca33ca19bd97c55c52eb3ca15901f27ade046cc79e69e87
+
+build:
+  number: 0
+  skip: true  # [win]
+  run_exports:
+    # The ABI is very stable
+    - {{ pin_subpackage('libxcrypt', max_pin=None) }}
+  script_env:
+    - OBSOLETE_API=no
+  script: $RECIPE_DIR/do_build.sh
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - make
+    - pkg-config
+    - perl
```

Comment:
This is going to create a cyclic graph.

Reply:
Is there anything we can/should do about it?

Reply:
You could use `perl` from yum.